build-common/NHibernate.props

```diff
@@ -16,6 +16,7 @@
 
     <NhAppTargetFrameworks Condition ="$(NhAppTargetFrameworks) == ''">net48;net6.0</NhAppTargetFrameworks>
     <NhLibTargetFrameworks Condition ="$(NhLibTargetFrameworks) == ''">net461;net48;netcoreapp2.0;netstandard2.0;netstandard2.1;net6.0</NhLibTargetFrameworks>
+    <RuntimeFrameworkVersion Condition="'$(TargetFramework)' == 'netcoreapp2.0'">2.0.3</RuntimeFrameworkVersion>
```

Comment:
That is the minimal non vulnerable version of netcoreapp2.0.

The main effect of this is to cause the vulnerability warning to go away. It may prevent usage of the library with a lower vulnerable version, but that is not clear if that applies to libraries or only applications. See [here][https://learn.microsoft.com/en-us/dotnet/core/versions/selection#self-contained-deployments-include-the-selected-runtime).

> For framework-dependent applications, the RuntimeFrameworkVersion specifies the minimum required runtime framework version.

Reply:
All version of .NET Core App 2.0 runtime are vulnerable. But for some reason not everything triggers a warning.